README.md

```diff
@@ -18,6 +18,20 @@ in AWS.
 
 ## Installing cloudlift
 
+### Using Script
+
+```sh
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/GetSimpl/cloudlift/master/scripts/installer.sh)"
+```
+
+Note: Append `sudo` to the above command if you are not running as root user. For running in a docker container, no need to append `sudo`.
+
```

Comment:
Remove the `Note:` please, since we do not necessary run the command with sudo, https://github.com/GetSimpl/cloudlift/pull/153/commits/bbceaa5458e636ee7cb2db4e18c97298630ee079 introduces implicit sudo